README.md

```diff
@@ -15,22 +14,40 @@ Read [the documentation here](https://imitation.readthedocs.io/en/latest/).
 
 ### Installing PyPI release
 
+Installing the PyPI release is the standard way to use `imitation`, and the recommended way for most users.
```

Comment:
Right now the PyPI release is very out of date... but that's something we should fix by making a release, not having people install from source, of course.

Reply:
Agreed. Do we currently have a stable version that we could release? Happy to help with this, never used PyPI and would be nice to learn.

Reply:
Thanks for the prod. I've set up an automation workflow to make it easier for us to do releases in the future and prepared a PR for v0.3 release: https://github.com/HumanCompatibleAI/imitation/pull/489 https://github.com/HumanCompatibleAI/imitation/pull/490

Once reviewed we should be able to make the release.